README.rst

```diff
@@ -324,6 +312,23 @@ If you just want to start a single instance of the app and run it directly::
     python -m synapse.app.homeserver --config-path homeserver.yaml
 
 
+Running the unit tests
+======================
+
+After getting up and running, you may wish to run Synapse's unit tests to
+check that everything is installed correctly::
+
+    python -m twisted.trial tests
+
+This should end with a 'PASSED' result (note that exact numbers will
+differ)::
+
+    Ran 1337 tests in 716.064s
+
+    PASSED (skips=15, successes=1322)
+
+For more tips on running the unit tests, like running a specific test or
+show the logger output of the app, see the `CONTRIBUTING doc <CONTRIBUTING.md#run-the-unit-tests>`_.
```

Comment:
> Running the tests is also covered in CONTRIBUTING.md, where I think this fits more naturally. Possibly put the extra info there with a "see [contributing] for more info" link?

For reference, I never checked out the contributing doc for tests because the readme already had content in it. I was only linked to it from the merge request template for changelog and linting stuff.

Hopefully the "see contributing doc" link is obvious enough. Good as a first iteration to have this documented somewhere ⏩

Reply:
:+1: